sql-odbc/src/PowerBIConnector/OdfeSqlOdbcPBIConnector.pq

```diff
@@ -106,6 +106,20 @@ OdfeSqlOdbcPBIConnectorImpl = (Host as text, optional Port as number) as table =
                     Value.ReplaceType(toTable, Value.Type(source))
             else
                 source,
+        
+        SQLGetFunctions = Diagnostics.LogValue("SQLGetFunctions_Options", [
+            SQL_API_SQLBINDPARAMETER = false
+        ]),
+
+        SqlCapabilities = Diagnostics.LogValue("SqlCapabilities_Options", [
+            SupportsTop = false,
+            LimitClauseKind = LimitClauseKind.LimitOffset,
+            SupportsNumericLiterals = true,
+            SupportsStringLiterals = true,
+            SupportsOdbcDateLiterals = true,
```

Comment:
Can you confirm these options for date/time/timestamp literals? Wondering if this refers to the `{dt ...}`-type syntax we've seen before, which ODFE SQL plugin doesn't support

Reply:
Yes, this refers to `{d ..}` syntax. But if we don't set this to true then date values will always be specified using Parameter Binding (https://docs.microsoft.com/en-us/power-query/odbc) which is also not supported.

I will check if it is possible to report the escape sequence syntax unsupported.

Reply:
ODBC driver sets `SQL_DATETIME_LITERALS` to 0 in SQLGetInfo and a value of "0" means that SQL-92 datetime literals are not supported. (https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqlgetinfo-function?view=sql-server-ver15)

Reply:
Verified with @rupal-bq that timestamp values work with the current PBI connector; will look back at this when we include DirectQuery support